cfgov/unprocessed/css/molecules/social-media.less

```diff
@@ -61,17 +61,17 @@
         }
     }
 
-    .list {
+    .m-list {
```

Comment:
It does use `m-list__horizontal' in the markup. This is used to inline-block the whole list, not just the list items, so that this works on one line:
![screen shot 2017-07-05 at 5 21 30 pm](https://user-images.githubusercontent.com/704760/27885569-6ebd5286-61a6-11e7-8a31-8b790576dbcb.png)